src/CMakeLists.txt

```diff
@@ -64,6 +65,10 @@ if(UMF_BUILD_CUDA_PROVIDER)
         ${cuda-headers_SOURCE_DIR}
         CACHE PATH "Path to CUDA headers")
```

Comment:
and same here, `CACHE` and `PATH "..."` would be redundant now...?

Reply:
Done

Reply:
Reverted, it does not work without it.